transfer-hbar/script-transfer-hbar.js

```diff
@@ -115,7 +115,7 @@ async function scriptTransferHbar() {
     const transferJsonAccountTransfersFinalAmounts = transferJsonAccountTransfers
         ?.map((obj) => Hbar.from(obj.amount, HbarUnit.Tinybar).toString(HbarUnit.Hbar));
     logger.log(
-        'The debit and credit amounts of the transfer transaction:\n',
+        'The debit, credit, and transaction fee amounts of the transfer transaction:\n',
```

Comment:
FYI @theekrystallee is looking into how to get this mapped out to each account, we can display a clearer representation here.
https://swirldslabs.slack.com/archives/C076TRA61JS/p1722568520115449?thread_ts=1722567923.867689&cid=C076TRA61JS
![image](https://github.com/user-attachments/assets/7cd322d2-9f89-4285-921f-dd8f5d4fcc72)

Reply:
So I think we have an answer we can work based off of:

> They greyed out is fee accounts, 800, 98 and node IDs. Which are the accounts fees go into.

So we could simply filter out `accountId < 50 || accountId  = 98 || accountId = 800 ` and print the rest.
Or print all of them, but label fee recipients.